_news/2021-07-23-welcome-no-borders-trans-safety-network.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: article
+author: Kim, Mallory & Iona
+title: No Borders Manchester & Trans Safety Network join Resistance Lab
+title_fancy: "No Borders Manchester &<br> <span>Trans Safety Network</span><br> join Resistance Lab"
+excerpt: By working with No Borders Manchester and Trans Safety Network we further expand our capacity to cover state violence and human rights injustices in Manchester and the UK. Both these groups bring unique perspectives of working with different structurally disadvantaged people and we look forward to producing formal outputs together.
+---
+
+Resistance Lab began as a simple provocation: can technology be used to help fight state violence, especially police brutality? 
+
+Beginning in 2018, representatives from Northern Police Monitoring Project and Geeks for Social Change facilitated a series of discussions between activists, campaigners, researchers, youth workers, academics and local residents. Over the first year our coalition grew to include both many individual members and group membership by Race, Roots and Resistance; The African Pot project; Kids of Colour; and Sites of Resistance.
+
+In August last year, we published our inaugural report [_A Growing Threat to Life: Taser usage by Greater Manchester Police_](/our-work/gmp-taser-update/index.html), which was then followed by a [report into the false claims made by GMP this year](/our-work/gmp-taser-update/index.html), and a [panel discussion on the use of Tasers nationally](/news/2021-02-25-no-more-tasers-panel.html). 
+
+Through this work we've been working closely with representatives from No Borders Manchester and Trans Safety Network who we are now delighted to announce are now formal group members of the Resistance Lab collective.xec
```

Comment:
what's with the `.xec` on the end?

Reply:
Ugh cut and pasted through a few too many editors!